yarn-project/noir-contracts/src/contracts/docs_example_contract/src/storage/locked.nr

```diff
@@ -0,0 +1,11 @@
+// docs:start:state_vars-PublicState
+use dep::aztec::state_vars::public_state::PublicState;
+use dep::aztec::types::type_serialisation::bool_serialisation::{
+    BoolSerialisationMethods,
+    BOOL_SERIALISED_LEN,
+};
+
+fn init(storage_slot: Field) -> PublicState<bool, BOOL_SERIALISED_LEN> {
+    PublicState::new(storage_slot, BoolSerialisationMethods)
+}
```

Comment:
I separate them into different files because it is easier for people to see what to import and how to new a state variable when they are all together without the distraction of other state variables. But agree that it might make the reader think this is the recommended pattern. I'll move them back to storage.nr 🥲